auth0/asyncify.py

```diff
@@ -21,6 +22,17 @@ def asyncify(cls):
         if callable(getattr(cls, func)) and not func.startswith("_")
     ]
 
+    class BareAsyncClient(cls):
```

Comment:
Thanks for this fix @cgearing! Would you mind just calling this `UsersAsyncClient` and I'd be happy to approve the PR

Reply:
Done! Sorry I missed your comment!

Reply:
Thanks @cgearing!